python/ql/test/library-tests/taint/config/TestNode.ql

```diff
@@ -0,0 +1,10 @@
+import python
+import semmle.python.security.TaintTracking
+import semmle.python.dataflow.Implementation
+import TaintLib
+
+
+from TaintTrackingNode n
+where n.getConfiguration() instanceof TestConfig
+select n.getTaintKind(), n.getLocation().toString(), n.getNode().toString(), n.getPath().toString(), n.getContext().toString()
```

Comment:
I think this would be better if the first column was the location. Currently, it's quite difficult to match up the rows with the various lines of code.

Reply:
done